bots/naughty/rhel-7-4/6960-storaged-regression-lvm-resize

```diff
@@ -0,0 +1,5 @@
+  File "/build/cockpit/bots/../test/verify/check-storage-resize", line *, in testResize
+*
+Error resizing logical volume: Command-line `lvresize "TEST"/"vol" -L 209715200b -r' exited with non-zero exit status 5:
+stdout: `Do you want to unmount "/run/media/admin/FSYS"? [Y|n] n
+*
```

Comment:
This is not a known issue with the operating system, unless we file a bug for it. If it's a cockpit thing, we can either workaround the bug in the tests, or fix the bug in Cockpit.